### DISCUSSION TITLE
How to set up white labeling for React Native (react-native-cli) example

### DISCUSSION THREAD

**Discussion by MbuthiaWaKihara** [codemagic-ci-cd/codemagic-docs#2858] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

I am looking for an example on how to set up white labeling using Codemagic for a React Native project. The detailed example in Codemagic documentation, and on YouTube, uses Flutter. As someone who is new to Codemagic, and unfamiliar with Flutter, it's pretty difficult to follow

### Expected results

React Native white labeling example

### Actual results

No examples found

**Comment by dam-ease** [comment]
Hello @MbuthiaWaKihara,
Setting up white-labeling for a React Native project using Codemagic involves configuring the project to support multiple brands or configurations and automating the build process for these configurations in Codemagic. While the available documentations are more focused on Flutter, the same principles apply to React Native. Do you have your react native project configured for white labelling? Kindly follow the steps as described [here](https://docs.codemagic.io/yaml-quick-start/white-label-getting-started/) and in cases you have flutter commands, they can be replaced with the corresponding react native commands. If there are any specific steps you'd like me to assist with, kindly share here and I'd be glad to help

**Comment by MbuthiaWaKihara** [comment, marked as answer]
I was able to follow the Flutter example, and I applied the changes to my React Native project. I however still wish there was a guide for all supported platforms.

**Reply by dam-ease** [reply]
Hey @MbuthiaWaKihara,
Glad to hear! I opened an issue for this already yesterday https://github.com/codemagic-ci-cd/codemagic-docs/issues/2861. You can also contribute to the (Sample Projects Repo)[https://github.com/codemagic-ci-cd/codemagic-sample-projects] and raise a PR.
